### ISSUE TITLE
Update the API Specs doc for L2 Domain/L2 Domain Template Name string length

### ISSUE THREAD

**Issue by linuxus** [nuagenetworks/vsd-api-specifications#47]
Please update the API specs doc to reflect the correct value for L2 Domain/L2 Domain Template Name field string length limit to be 64 characters rather than 255.

![image](https://user-images.githubusercontent.com/3790483/32349179-92885194-bfec-11e7-99c0-96dc9f8713a0.png)

Since VSD UI only allows 64 char.

Thanks.

**Comment by hellt** [comment]
This is actual backend report

```
{
    "errors": [
        {
            "property": "name",
            "descriptions": [
                {
                    "description": "The length of the input should be between 1 and 64 character(s).",
                    "title": "Invalid length"
                }
            ]
        }
    ],
    "internalErrorCode": 5000
}
```

**Comment by pdellaert** [comment]
Fixed in 5.2.1
